src/apartments/__mocks__/entities/apartment-entity.mock.ts

```diff
@@ -0,0 +1,13 @@
+import { faker } from '@faker-js/faker';
+
+import { houseEntity } from '../../../houses/__mocks__/entities/house.entity.mock';
+import { ApartmentEntity } from '../../entities/apartment.entity';
+
+const { number } = faker.datatype;
+
+export const apartmentEntity = {
+  id: number(),
+  number: number(),
+  floor: number(),
+  house: houseEntity,
+} as ApartmentEntity;
```

Comment:
I believe, that in the mocks you want to have a predefined set of items and later in `repository mock` you simply do something like:

```typescript
const mockItems = [{ id: 1 }, { id: 2 }, { id: 3 }];

const mockRepository = {
  findOne: jest.fn((options) => mockItems.find(i => i.id === options.id))
}
```